cdk/script/ci

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+nvm_available() {
+  type -t nvm > /dev/null
+}
+
+source_nvm() {
+  if ! nvm_available; then
+    [ -e "/usr/local/opt/nvm/nvm.sh" ] && source /usr/local/opt/nvm/nvm.sh
+  fi
+  if ! nvm_available; then
+    [ -e "$HOME/.nvm/nvm.sh" ] && source "$HOME/.nvm/nvm.sh"
+  fi
+}
+
+source_nvm
+nvm install
+nvm use
+
+npm install -g yarn
```

Comment:
Can we use the `guardian/actions-setup-node@v2.4.1` in the workflow to provide a node version from the `.nvmrc` file auto-magically instead of this `nvm` dance? It also comes with `yarn` so we don't have to globally install it.

https://github.com/guardian/.github/blob/main/workflow-templates/ci-sbt-node.yml#L34 might be helpful.

Reply:
Ah great, yep was copying from Amigo but this is nicer.